### ISSUE TITLE
Broken links in https://lbry.tech/contribute

### ISSUE THREAD

**Issue by apoorva-mk** [lbryio/lbry.tech#305]
Other than the first link to "Ecosystem Overview" all others seem to be broken.

**Comment by apoorva-mk** [comment]
I would like to work on this issue.

**Comment by tzarebczan** [comment]
There may be a pr to fix this already. Which link in particular?

**Comment by tzarebczan** [comment]
Looks like it's still broken, feel free to submit a PR.

**Comment by apoorva-mk** [comment]
Thanks I will send a PR immediately @tzarebczan

**Comment by dougwebb** [comment]
It looks like there are still broken links. For example, https://lbry.com/bounty/lbry-binding (which is on the https://lbry.tech/resources/api-wrappers page) redirects to https://lbry.tech/contribute but there's nothing there about developing new bindings.

Also, https://lbry.tech/api/sdk just sits in a loop reloading itself. Tested on both Firefox and Chrome. No error messages showing up in the console. Something in the javascript is making the page refresh.

**Comment by tzarebczan** [comment]
Thanks Doug - do you wanna take a shot at fixing them? Would be much appreciated!

**Comment by dougwebb** [comment]
I would if I could, but I'm looking around in the lbry.tech repo and I
can't find the API docs or bounty page, so I have no idea what the fixes
would be.
I did find https://github.com/lbryio/lbry-sdk/blob/master/docs/api.json
which appears to be the source for the API reference documentation, but I'm
not familiar with tools for turning that into a webpage. I'd also
expect/hope to see tutorial/overview style documentation to go along with
the auto-generated reference documentation.

For what it's worth, I'd be interested in contributing a C# binding for the
API. I've written C# wrappers for a bunch of REST/JSON apis for electronic
payments, so I could get a lbry wrapper put together easily. If api.json is
authoritative, I'd also look at using it to auto-generate the C# wrapper
code so it's easy to update after API changes.

Doug.

**Comment by tzarebczan** [comment]
We've updated the api links, should work now. The bounty page is gone and should be removed.

**Comment by misspellted** [comment]
I was reviewing the Contributor's Guide, and it looks as if there are still a couple of broken links trying to connect to the Ecosystem Overview:

* contribute.md#reporting-a-bug
* contribute.md#feature-requests

After following the procedure for Reporting a Bug, I discovered that @apoorva-mk had already created the issue, but I do not see a follow-up PR addressing the 2 broken Ecosystem Overview links noted above.

Additionally, I was unable to find the template for reporting a bug for this repository. Should that be a separate issue? I don't know if the following is sufficient (it is a format I've used in prior roles), but to expand on the initial report by @apoorva-mk:

**Expected Behavior**
Clicking on the ecosystem overview links under the Reporting a Bug or Feature Requests sections of the Contributor's Guide takes the viewer to the Ecosystem Overview diagram in the Overview article that the Ecosystem Overview link does in the Repository Overview section.

**Actual Behavior**
Clicking on the ecosystem overview links under the Reporting a Bug or Feature Requests sections of the Contributor's Guide does not take the viewer to the Ecosystem Overview diagram in the Overview article.

**Observational Environment**
Platform: Pop!_OS 19.10
Browser: Firefox 72.0.1 (64-bit)

**Steps to Reproduce**
Open the Contributor's Guide in a browser.
Scroll to the Repository Overview section.
Click on the Ecosystem Overview link in the Repository Overview section.
Observe the Ecosystem Overview diagram in the Overview article opened in the same tab.
Return to the Contributor's Guide using the browser's Back navigation.
Scroll to the Report a Bug section.
Click on the ecosystem overview link in the Repository Overview section.
Observe the lack of the browser navigating to the Overview article to show the Ecosystem Overview diagram.
Scroll to the Features Requests section.
Click on the ecosystem overview link in the Repository Overview section.
Observe the lack of the browser navigating to the Overview article to show the Ecosystem Overview diagram.
Scroll slightly up to the appreciation link at the bottom of the Report a Bug section.
Click the appreciation link.
Observe the browser scroll down to the Appreciation section of the Contributor's Guide.

**Demonstration**
And if the StR portion is too much to read, I captured it in a short video: lbry://@misspellted#a/some-contrib-guide-eco-view-links-broken#1
